.github/workflows/js.yml

```diff
@@ -61,6 +62,7 @@ jobs:
     - name: Build JS SDK
       run: tools/bin/mage jsSDK:clean jsSDK:build
     - name: Install JS dependencies
+      if: steps.yarn-cache.outputs.cache-hit != 'true'
```

Comment:
Would it make sense to do the same for `jsSDK:deps` above?

Reply:
Same in other files

Reply:
From testing various scenarios there is not much improvement when checking for cache hit for js sdk dependencies. It also seems that there are some issues in js/js sdk mage targets that cause weird diff issues in CI in some cases. Ill investigate this further in a separate PR together with improvements for `js:build` target.

I added yarn cache check to `release-snapshot` and `general` workflows as well.